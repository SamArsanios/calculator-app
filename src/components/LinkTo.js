import React from 'react';
import PropTypes from 'prop-types';

const LinkTo = props => {
    const { name, path } = props;
    return (
        <a className="m-left p-left" href={path}>{name}</a>
    );
};

LinkTo.PropTypes = {
    name: PropTypes.string.isRequired,
    path: PropTypes.string.isRequired
}

export default LinkTo;