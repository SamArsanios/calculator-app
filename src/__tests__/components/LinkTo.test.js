import { render } from '@testing-library/react';
import React from 'react';
import renderer from 'react-test-renderer';
import LinkTo from '../../components/LinkTo';

describe('LinkTo', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<LinkTo />)
      .toJSON()
    expect(tree).toMatchSnapshot();
  });
});