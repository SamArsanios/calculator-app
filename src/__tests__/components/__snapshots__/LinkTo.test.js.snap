// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinkTo renders correctly 1`] = `
<a
  className="m-left p-left"
/>
`;
